src/VisualStudio/Core/Impl/RoslynVisualStudioWorkspace.cs

```diff
@@ -172,8 +172,7 @@ public override bool TryGoToDefinition(ISymbol symbol, Project project, Cancella
             }
 
             return GoToDefinitionHelpers.TryGoToDefinition(
-                searchSymbol, searchProject, _navigableItemsPresenters,
-                containingTypeSymbol: null, throwOnHiddenDefinition: false, cancellationToken: cancellationToken);
+                searchSymbol, searchProject, _navigableItemsPresenters, cancellationToken: cancellationToken);
```

Comment:
There's TryGoToDefinition and TryFindAllReferences APIs here. Why not add a TryGoToImplementation?